contracts/Governance/GovernorBravoInterfaces.sol

```diff
@@ -68,15 +68,6 @@ contract GovernorBravoDelegatorStorage {
  */
 contract GovernorBravoDelegateStorageV1 is GovernorBravoDelegatorStorage {
 
-    /// @notice The delay before voting on a proposal may take place, once proposed, in blocks
-    uint public votingDelay;
-
-    /// @notice The duration of voting on a proposal, in blocks
-    uint public votingPeriod;
-
-    /// @notice The number of votes required in order for a voter to become a proposer
-    uint public proposalThreshold;
-
```

Comment:
We can't delete storage entries, we can only deprecate them. I propose we keep these as private vars with a deprecation warning, see e.g. https://github.com/VenusProtocol/venus-protocol/blob/a23f0fa7dc90bfdae2ebf09e82a72a0665d9f42b/contracts/XVSVault/XVSVaultStorage.sol#L83-L87

Reply:
My bad... I was in context of isolated pools, so forgot we are doing incremental upgrade here